server/config.ts

```diff
@@ -106,14 +106,6 @@ export default {
       },
       agent: DEFAULT_AGENT_CONFIG,
     },
-    frontendComponents: {
-      url: get('COMPONENT_API_URL', 'http://localhost:8082', requiredInProduction),
-      timeout: {
-        response: Number(get('COMPONENT_API_TIMEOUT_SECONDS', 2000)),
-        deadline: Number(get('COMPONENT_API_TIMEOUT_SECONDS', 2000)),
-      },
-      agent: DEFAULT_AGENT_CONFIG,
-    },
```

Comment:
We still need `COMPONENT_API_URL`  providing in the helm config, but the library retrieves that configuration itself without needing to define it here.